tests/test_filter_jobs.py

```diff
@@ -11,10 +11,16 @@
 def test_filter_jobs(base_url, selenium):
     """Open resultset page and filter for platform"""
     page = TreeherderPage(selenium, base_url).open()
-    platform = u'Linux'
+    all_builds = page.all_builds
 
+    platform = u'Linux'
     page.filter_by(platform)
-    assert platform in page.result_sets[0].builds[0].platform_name
+    filtered_builds = page.all_builds
+
+    buildname = page.result_sets[0].builds[0].platform_name
+    assert platform in buildname
```

Comment:
Let's just do `assert platform in page.result_sets[0].builds[0].platform_name` and avoid the `buildname` variable.